module/VuFind/src/VuFind/AjaxHandler/GetRecordCover.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * GetRecordCover AJAX handler.
```

Comment:
Is this handler being used yet, or just work in progress? I couldn't spot a place where it comes into play yet, but maybe I wasn't looking closely enough.

Reply:
Ajax handler is WIP, not finished yet, I am going to work on it as a next step.

Reply:
Sounds good -- just let me know whenever you're ready for another review!